docs/docs/tuning-your-model.mdx

```diff
@@ -434,9 +434,48 @@ underlying tensorflow ops `tf.sparse.sparse_dense_matmul`,
 `tf.nn.sparse_softmax_cross_entropy_with_logits`,
 and `tf.math.unsorted_segment` ops. For more information see [here](https://github.com/tensorflow/community/blob/master/rfcs/20210119-determinism.md)
 
-You can see which featurizers are sparse [here](./components.mdx#featurizers),
+For the above reasons, the models are also not guaranteed to yield the exact same
+performance when trained on a GPU across multiple runs. This even applies to situations where training is run
+multiple times on the same training data with the same config
+and random seeds set appropriately while being evaluated on a standard held-out test set.
+Internal experiments have shown the following fluctuation in model's performance when trained and evaluated on a held-out
+test set of a variety of datasets (experiments were run 5 times on each dataset):
+
+```
++-----------------------+------------+------------------+------------------+-------------------+
+| Task,                 | Number of  | Average standard | Minimum standard | Maximum standard  |
+| Metric                | datasets   | deviation        | deviation        | deviation         |
++-----------------------+------------+------------------+------------------+-------------------+
+| Intent Classification | 11         | 0.0042           | 2.4e-5           | 0.0176            |
+| Macro F1              |            |                  |                  |                   |
++-----------------------+------------+------------------+------------------+-------------------+
+| Entity Recognition    | 7          | 0.0019           | 0.0007           | 0.0044            |
+| Macro F1              |            |                  |                  |                   |
++-----------------------+------------+------------------+------------------+-------------------+
+| Response Selection    | 2          | 0.0098           | 0.0003           | 0.0231            |
+| Macro F1              |            |                  |                  |                   |
++-----------------------+------------+------------------+------------------+-------------------+
+| Action Selection      | 5          | 0.0025           | 0.0010           | 0.0053            |
```

Comment:
Is `Action Selection` referring to predictions from TED Policy?

Reply:
The task refers to predictions from the complete ensemble of policies but `TEDPolicy` is the only ML based policy which can result in fluctuations in results.